include/cst_tokenstream.h

```diff
@@ -77,7 +77,11 @@ typedef struct cst_tokenstream_struct {
     const cst_string *p_prepunctuationsymbols;
     const cst_string *p_postpunctuationsymbols;
 
+
     cst_string charclass[256];
+    cst_string *charclass2[256];
+    cst_string **charclass3[256];
+    cst_string ***charclass4[256];
 
```

Comment:
More than a performance vs memory, I chose to use 256 so it is readable that each charclass matches possible bytes values. I will try to make these arrays as short as needed, I don't think the impact on performance is significant.